doc/src/modules/solvers/solveset.rst

```diff
@@ -315,19 +317,19 @@ What are the general methods employed by solveset to solve an equation?
    used as follows:
 
    - The method to solve the rational function is called:
-     ``_solve_as_rational(f, symbol, solveset_solver, as_poly_solver)``
-     its third argument is the ``solveset_solver`` which can either be
-     ``solveset_real`` or ``solveset_complex`` based on these, respective
-     poly solvers ``_solve_as_poly_real`` or ``_solve_as_poly_complex``
-     is called to solve as polynomial.
+     ``_solve_as_rational(f, symbol, solveset_solver, as_poly_solver)`` its
```

Comment:
actually, I would leave off the args and just say that `_solve_as_rational` is used